config/processCcd.py

```diff
@@ -26,9 +26,10 @@
     print "WARNING: Unable to use psfex: %s" % e
     config.charImage.measurePsf.psfDeterminer.name = "pca"
 
+if True:
     # we don't have astrometry_net data (yet) so astrom and photo cal are impossible
     config.doCalibrate=False
-else:
+elif False:
```

Comment:
It's a convenience for the user.  A dirty convenience I grant you, but the whole package is dirty...